amazon_transcribe/client.py

```diff
@@ -84,6 +84,10 @@ async def start_stream_transcription(
         enable_partial_results_stabilization: Optional[bool] = None,
         partial_results_stability: Optional[str] = None,
         language_model_name: Optional[str] = None,
+        identify_language: Optional[bool] = False,
+        preferred_language: Optional[str] = None,
+        identify_multiple_languages=False,
+        language_options=None,
```

Comment:
I am not familiar with AWS coding standards, but is there a reason why there isn't type hiting on `identify_multiple_languages` and `language_options` ?